test/acceptance/behave/requirements.txt

```diff
@@ -6,4 +6,4 @@ requests==2.7.0
 xmltodict==0.9.2
 fabric==1.10.2
 pymongo==3.0.3
-git+https://visitor:visitor@pdihub.hi.inet/fiware/fiware-iotqaUtils@develop
\ No newline at end of file
+git+https://github.com/telefonicaid/iot-qa-tools.git
```

Comment:
these libraries are changing with each feature in CB, really is a "moving picture". This repo is used to unify libraries with the integration team.